src/fn/upload-app-settings.js

```diff
@@ -1,14 +1,24 @@
 const api = require('../lib/api');
-const environment = require('../lib/environment');
 const fs = require('../lib/sync-fs');
+const skipFn = require('../lib/skip-fn');
+const { info, error } = require('../lib/log');
+
+module.exports = (projectDir, couchUrl) => {
```

Comment:
This data `(projectDir, couchUrl)` is no longer passed into the action and is instead accessed through `environment.pathToProject`. Sorry for that breaking change.

Would you update to use the environment.pathToProject interface to point to the project's directory?
I think the `skip-fn` file is actually no longer used in the codebase anywhere - would you mind deleting the file?
Can you give this a quick test?

Reply:
All done!